single-pool/program/src/error.rs

```diff
@@ -55,8 +55,8 @@ pub enum SinglePoolError {
     #[error("SignatureMissing")]
     SignatureMissing,
     /// Stake account is not in the state expected by the program.
-    #[error("WrongStakeState")]
-    WrongStakeState,
+    #[error("WrongStakeStateV2")]
+    WrongStakeStateV2,
```

Comment:
```suggestion
    #[error("WrongStakeState")]
    WrongStakeState,
```
overzealous search/replace?